virttest/gluster.py

```diff
@@ -204,80 +201,6 @@ def gluster_brick_delete(brick_path, session=None):
             logging.error("Not able to delete brick folder %s", details)
 
 
-@error_context.context_aware
-def gluster_vol_create(vol_name, hostname, brick_path, force=False, session=None):
```

Comment:
If you just want to avoid circular imports, it's better to look for calling alternatives rather than move these things out

Reply:
I could not get about this idea, can you help me to explain it ?